src/index/types/markdown.ts

```diff
@@ -601,6 +608,13 @@ export class MarkdownListItem implements Indexable, Linkbearing, Taggable, Field
         return MarkdownListItem.FIELD_DEF(this);
     }
 
+    /** return text without annotations + indentation */
+    get $strippedText() {
```

Comment:
I'd probably call this something like `$cleantext` or similar since stripped is somewhat technical.